app/views/partials/_comments.html.erb

```diff
@@ -1,7 +1,7 @@
-<div class="card bg-success">
-  <div class="card-body">
+<%= stylesheet_link_tag 'post', media: 'all', 'data-turbolinks-track': 'reload' %>
+<div id="comment">
     <% comments.each do |comment| %>
-      <p><strong><%= comment.author.name %></strong>: <%= comment.text %></p>
+      <p class="commentD"><strong><%= comment.author.name %></strong>: <%= comment.text %></p>
     <% end %>
   </div>
-</div>
+
```

Comment:
![image](https://github.com/AleWaweru/blog-App/assets/93039359/4f0e8935-e2a7-438a-8c87-8662d688e4c6)

- [ ] I can see that you have renamed the Comments Counter and Likes Counter so kindly rename them everywhere you have used them in models and views 👍 to make your app work as expected 😊

Reply:
@HAMMAS-SALEEM  done